site/documentation/content/api/command-and-control-pubsub/index.md

```diff
@@ -0,0 +1,178 @@
+| *deviceId*       |    yes    | *string*  | The identifier of the device that sent the response.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
+| *status*         |    yes    | *integer* | MUST indicate the status of the execution. See table below for possible values.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
+| *tenantId*       |    yes    | *string*  | The identifier of the tenant that the device belongs to.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
+| *content-type*   |    no     | *string*  | If present, MUST contain a *Media Type* as defined by [RFC 2046](https://tools.ietf.org/html/rfc2046) which describes the semantics and format of the command's input data contained in the message payload. However, not all protocol adapters will support this property as not all transport protocols provide means to convey this information, e.g. MQTT 3.1.1 has no notion of message headers.<br>If the response is an error message sent by the Hono protocol adapter or Command Router component, the content type MUST be *application/vnd.eclipse-hono-delivery-failure-notification+json*.   |
```

Comment:
can we also please add *delivery-failure-notification-metadata* as defined by the Kafka based API?